utils/roll-deps

```diff
@@ -14,22 +14,44 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Attempts to roll all entries in DEPS to origin/master and creates a
+# Attempts to roll all entries in DEPS to origin/master and create a
```

Comment:
What about: Attempts to roll all entries in DEPS to tip-of-tree and create a commit

Reply:
Forgot to update the comment.